json-schemas/json

```diff
@@ -0,0 +1,535 @@
+mod-circulation-storage/ramls/staff-slip.json
```

Comment:
do these files (json and metadata) need file extensions?  .txt? I found it unexpected that "json" was not a json file. or maybe a name change? "source_json" or something.

Reply:
If it's confusing, I'll just throw the whole thing away. It's just a list of the files we're expecting to sync.

Reply:
Ah so nothing is functionally using or reading from this file? Seems OK to throw out then.